### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by dallemang** [edmcouncil/fibo#1577]
The published files prod.fibo.quickstart and dev.fibo.quickstart do not include the build version information (e.g., 2021Q3).  As far as I can tell, this information isn't available in a queryable form anywhere in any publication, nor in github; it is only available as a part of a versionIRI in an import. 

If someone downloads a version of FIBO, and saves it elsewhere, it is difficult to determine which version it is.  

Can we include that information in the published files somehow?  Maybe by including some of the data from the file https://github.com/edmcouncil/fibo/blob/master/MetadataFIBO.rdf , and then adding the version information, perhaps using dct:hasVersion ?

**Comment by mereolog** [comment]
We will update https://github.com/edmcouncil/ontology-publisher as soon as possible.

**Comment by mereolog** [comment]
Will adding something like this be enough?

```
@prefix fibo-spec: <https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/> .
fibo-spec:  owl:versionIRI  <https://spec.edmcouncil.org/fibo/ontology/20201201/MetadataFIBO/> .
```

**Comment by dallemang** [comment]
I'd like the release string as well .... eg 2021Q3

**Comment by mereolog** [comment]
@dallemang Fair enough, but at the moment we do not store this kind of release information within FIBO ontologies, i.e., it is sort of external.
On the other hand, we can add quite easily the timestamp when quick FIBO versions were created, well perhaps when the ontology-publisher process started.

**Comment by rivettp** [comment]
It should be in the manifest each time we publish from Dev or Prod - we already have the long-standing issue for this in ontology-publisher.

**Comment by mereolog** [comment]
@rivettp what do you mean by the "manifest"? Something like the manifest file in JAVA?

**Comment by rivettp** [comment]
@mereolog yes a separate file. We already produce one but it's useless. We already have an issue for this we've discussed in Jira https://jira.edmcouncil.org/browse/INFRA-264?jql=text%20~%20%22manifest%22
